src/createRouter.ts

```diff
@@ -54,6 +55,8 @@ export interface RouterMixin {
 	 */
 	routes?: Route<Parameters>[];
 
+	prevHierarchy?: Selection[];
```

Comment:
I find "hierarchy" confusing. To me that's how the routes are structured, not which ones were last selected.